index.d.ts

```diff
@@ -781,13 +781,12 @@ declare module 'plaid' {
 
   interface SandboxItemSetVerificationStatusResponse extends BaseResponse {}
 
-  interface ClientOptions {
+  interface ClientOptions extends AxiosRequestConfig {
     version?: '2020-09-14' | '2019-05-29' | '2018-05-22' | '2017-03-08';
     clientApp?: string;
-    timeout?: number;
```

Comment:
This should no longer be necessary as it is duplicative of the `timeout` property in `AxiosRequestConfig` and in previous versions when typed correctly was not apart of it https://github.com/plaid/plaid-node/blob/5.2.0/index.d.ts#L720